ellipse/bin/main.ml

```diff
@@ -0,0 +1,22 @@
+  open Claudius
+
+  let tick (_t : int) (screen : Screen.t) (_prev : Framebuffer.t) (_inputs : Base.input_state) : Framebuffer.t =
+>>>>>>> File-updates
```

Comment:
You have a merge conflict line in here, this I imagine doesn't actually run - have you checked since you did the merge?